filers/default/src/main/java/org/commonjava/indy/filer/def/DefaultGalleyStorageProvider.java

```diff
@@ -205,12 +207,29 @@ private void setupCacheProviderFactory()
 
         if ( pathDB != null )
         {
-            String prefix = metricsManager.getConfig().getNodePrefix();
-            pathDB = new MeasuredPathDB( pathDB, metricsManager.getMetricRegistry(), getSupername( prefix, "pathDB" ) );
+            final IndyMetricsConfig metricsConfig = metricsManager.getConfig();
+            if ( metricsConfig.isPathDBMetricsEnabled() )
```

Comment:
A question here: if path-mapped-s cache provider is not enabled, will this metrics be enabled?

Reply:
No. if isStandalone, the code returns earlier, will not get here.